Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionAzureNSG.yaml

```diff
@@ -0,0 +1,138 @@
+  - Title: ASIM
+    Link: https://aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports normalizing Azure NSG logs to the ASIM Network Session normalized schema.
+ParserName: ASimNetworkSessionAzureNSG
+EquivalentBuiltInParser: _ASim_NetworkSession_AzureNSG
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let DvcActionLookup = datatable(FlowStatus_s:string, DvcAction:string, EventResult:string) [
+      'A', 'Allow', 'Success',
+      'D', 'Deny', 'Failure',
+  ];
+  let NetworkDirectionLookup = datatable(FlowDirection_s:string, NetworkDirection:string, isOutBound:bool) [
+      'I', 'Inbound', false,
+      'O', 'Outbound', true
+  ];
+  let NetworkProtocolLookup = datatable(L4Protocol_s:string, NetworkProtocol:string)[
+      'T', 'TCP',
+      'U', 'UDP'
+  ];
+  let parser = (disabled:bool=false) 
+  {
+    let AzureNetworkAnalytics = 
+      AzureNetworkAnalytics_CL
+      | where not(disabled) and isnotempty(FlowType_s)
+      | lookup NetworkDirectionLookup on FlowDirection_s
+    ;
+    let AzureNetworkAnalyticsInbound =
+      AzureNetworkAnalytics
+      | where not(isOutBound)
+      | project-rename
+          DstMacAddr = MACAddress_s
+      | extend
+          DstBytes = tolong(OutboundBytes_d), // -- size fields seem not to be populated for inbound
+          DstPackets = tolong(OutboundPackets_d),
+          SrcBytes = tolong(InboundBytes_d),
+          SrcPackets = tolong(InboundPackets_d),
+          SrcInterfaceName = split(NIC_s, '/')[1],
+          SrcGeoCountry = toupper(Country_s)
+      | extend hostelements=split(VM2_s,'/')
+      | extend 
+          DstFQDN = strcat(hostelements[0], @"\", hostelements[1]),
+          DstHostname = tostring(hostelements[1]),
+          DstDomain = tostring(hostelements[0]),
+          DstDomainType = "ResourceGroup"
+      | extend Hostname = DstHostname
+      | project-away hostelements
+    ;  
+    let AzureNetworkAnalyticsOutbound =
+      AzureNetworkAnalytics
+      | where isOutBound
+      | project-rename
+          SrcMacAddr = MACAddress_s
+      | extend
+          SrcBytes = tolong(OutboundBytes_d), 
+          SrcPackets = tolong(OutboundPackets_d),
+          DstBytes = tolong(InboundBytes_d),
+          DstPackets = tolong(InboundPackets_d),
+          DstInterfaceName = split(NIC_s, '/')[1],
+          DstGeoCountry = toupper(Country_s)
+      | extend hostelements=split(VM1_s,'/')
+      | extend 
+          SrcFQDN = strcat(hostelements[0], @"\", hostelements[1]),
+          SrcHostname = tostring(hostelements[1]),
+          SrcDomain = tostring(hostelements[0]),
+          SrcDomainType = "ResourceGroup"
+      | extend Hostname = SrcHostname
+      | project-away hostelements
+    ;
+    union AzureNetworkAnalyticsInbound, AzureNetworkAnalyticsOutbound
+    | project-rename
+        Dvc = NSGList_s,
+        DvcSubscriptionId = Subscription_g,
+        EventEndTime = FlowEndTime_t,
```

Comment:
For some schemas docs say EventEndTime is alias to Time Generated. For nw the csv treats it as an alias.
Parser is ok, doc to be updated